build/scripts/restore.ps1

```diff
@@ -0,0 +1,13 @@
+param ([string]$nugetZipUrl = $(throw "Need an URL to the NuGet zip") )
+
+$destination = ${env:UserProfile}
+$outFilePath = [IO.Path]::ChangeExtension([IO.Path]::GetTempFileName(), "zip")
```

Comment:
It was necessary when I was using the shell to do the unzip.  Now i may be able to avoid it.